operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-secrets-config.yaml

```diff
@@ -67,14 +68,18 @@ spec:
               if [[ -n $SIG_KEY_DATA ]]; then
                 echo "Signing secret exists."
```

Comment:
technically, this is not precisely accurate ... the secret k8s object could exist, but it is missing its `data` field

Reply:
looks good now thanks